docs/config.yaml

```
@@ -270,6 +269,8 @@ clouds:
 
 # OPTIONAL: miscellaneous options
 misc:
+  # OPTIONAL: whether to use a versionary to derive version numbers
```

Is a `versionary` enough of a noun? 

```suggestion
  # OPTIONAL: whether to use a versionary script to derive version numbers
```

---

I'm trying to make it become one. :) But fine to make it less cool.